packages/website/docs/using-vue.md

```diff
@@ -0,0 +1,204 @@
+---
+id: using-vue
+title: Using Autocomplete with Vue
+---
+
+Learn how to embed Autocomplete into a Vue application.
+
+You can integrate an Autocomplete instance into a Vue application using [Vue's Composition API](https://v3.vuejs.org/guide/composition-api-introduction.html#why-composition-api). Specifically you can instantiate an Autocomplete instance in the [`onMounted`](https://v3.vuejs.org/api/composition-api.html#lifecycle-hooks) lifecycle hook in the [`setup`](https://v3.vuejs.org/guide/composition-api-setup.html) function.
```

Comment:
this is only using vue 3, while Vue InstantSearch for example still only supports v2, can we make this also for vue 2 as well?

Reply:
We might cover Vue 2 in another iteration but that's not a priority before release.

Reply:
makes sense I guess, but that means it won't work with Vue InstantSearch

Reply:
GTK, definitely something to include when we write an "integrating with IS" guide.